templates/taxi/driver_list.html

```diff
@@ -20,7 +26,8 @@ <h1>
     {% for driver in driver_list %}
       <tr>
         <td>{{ driver.id }}</td>
-        <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
+        <td><a href="{{ driver.get_absolute_url }}">
+          {{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
```

Comment:
The link to the driver's detail page uses `get_absolute_url`. Consider using the `url` template tag for consistency and clarity: `<a href="{% url 'taxi:driver-detail' pk=driver.id %}">`.